### ISSUE TITLE
Output JSON unreadable in Swagger UI

### ISSUE THREAD

**Issue by ajbozarth** [IBM/MAX-Image-Segmenter#3]
Due to the format of the JSON output, when using the Swagger UI the output causes a ridiculously long scroll. We should consider if there's a way to limit the size of the output box in the swagger UI so a user can scroll through the output box and not the whole page. This isn't a huge issue just a small usability one, so it can be ignored if everyone agrees as such.

Issue discovered during testing of #2

**Comment by bdwyer2** [comment]
https://github.com/swagger-api/swagger-ui/issues/4018

**Comment by gdequeiroz** [comment]
@bdwyer2 @ajbozarth Can we close this?

**Comment by bdwyer2** [comment]
I think this is okay to close. It's a low priority issue and we are blocked by swagger and it doesn't seem like they are making any progress.
